error: `name` and `path_name` can not be used together
 --> tests/ui/err/has-name-and-path-name.rs:3:1
  |
3 | #[trace(name = "Name", path_name = true)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `trace` (in Nightly builds, run with -Z macro-backtrace for more info)
